### ISSUE TITLE
Reformatting

### ISSUE THREAD

**Issue by 9fo** [Disitha395ra/PCC-Schoolwebsite#3]
U should use folders to format the website into making it simpler for u to understand. Just a tip of course!

**Comment by mdenjamulislam** [comment]
Right! You should use folders and make a better format.

**Comment by parasshinde** [comment]
Yes, It is very confusing to see the code, You should try a better format for better understanding
